app-service-certificate-standard/README.md

```diff
@@ -0,0 +1,10 @@
+By default, 'Microsoft.CertificateRegistration' and 'Microsoft.Web' RPs don't have access to the Key Vault specified in the template hence you need to authorize these RPs by executing 
```

Comment:
This template seems like it should be called 101-app-service-certificate-standard